homeassistant/components/media_player/__init__.py

```diff
@@ -1053,6 +1058,20 @@ def __init__(
         self.children = children
         self.children_media_class = children_media_class
         self.thumbnail = thumbnail
+        self.sortable_title = sortable_title
+
+    @property
+    def is_sortable(self):
```

Comment:
We shouldn't make any changes to the `BrowseMedia` class or this file. We should just sort the items inside Kodi before creating BrowseMedia instances.